stdlib/public/core/KeyPath.swift

```diff
@@ -1764,7 +1764,11 @@ internal struct KeyPathBuffer {
     internal mutating func pushRaw(size: Int, alignment: Int)
         -> UnsafeMutableRawBufferPointer {
       var baseAddress = buffer.baseAddress.unsafelyUnwrapped
+    #if os(Android) && arch(arm64)
+      var misalign = Int(bitPattern: baseAddress) & (alignment - 1)
+    #else
       var misalign = Int(bitPattern: baseAddress) % alignment
+    #endif
```

Comment:
@airspeedswift, rebased and made this KeyPath change I mentioned in my last comment, so this pull now only affects Android AArch64 and will have no effect on any other platform.